frontend/lib/about/dev_settings_view.dart

```diff
@@ -54,33 +54,25 @@ class DevSettingsView extends StatelessWidget {
       padding: const EdgeInsets.all(15.0),
       child: Column(
         children: [
-          ListTile(
-            title: const Text('Reset cards'),
-            onTap: () => _resetEakData(context, userCodeModel),
-          ),
-          ListTile(
-            title: const Text('Set (invalid) sample card'),
-            onTap: () => _setSampleCard(context),
-          ),
+          ListTile(title: const Text('Reset cards'), onTap: () => _resetEakData(context, userCodeModel)),
+          ListTile(title: const Text('Set (invalid) sample card'), onTap: () => _setSampleCard(context)),
```

Comment:
❓ tbh I think the previous formatting is better readable
not insisting though if its just my opinion